eth/api.go

```diff
@@ -1197,6 +1215,10 @@ func (s *PublicTransactionPoolAPI) GetTransactionReceipt(txHash common.Hash) (ma
 		fields["contractAddress"] = receipt.ContractAddress
 	}
 
+	if receipt.Status != types.TxStatusUnknown {
+		fields["status"] = receipt.Status
```

Comment:
- https://wiki.parity.io/JSONRPC-eth-module#eth_gettransactionreceipt
- https://github.com/paritytech/parity/blob/28c731881f2da0ceca4752dbcc1a8f9ad041f988/ethcore/types/src/receipt.rs#L62-L108

Continuing from sorpaas's [point 2.](https://github.com/ethereumproject/go-ethereum/pull/431#issuecomment-350511324), we need to be careful about changing established `eth_` namespaced APIs. So either we'll have to check block number, too, or create a new API method. I'd be keen to use his suggestion of `etc_`, or our `geth_` namespace. (Which one is a question worth talking moar about).

Reply:
Actually, we're not compatible anyways right now - because we're not returning `nil` - we're just skipping the `status` field.